Proton/Proton.xcodeproj/project.pbxproj

```diff
@@ -1165,6 +1296,58 @@
 			};
 			name = Release;
 		};
+		1BB4AAED2606DA2D000D7584 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CODE_SIGN_STYLE = Automatic;
+				DEFINES_MODULE = YES;
+				DEVELOPMENT_TEAM = FUJ76485CT;
```

Comment:
Probably, no need to lock `DEVELOPMENT_TEAM` I can build it locally without specifying the team
```suggestion
```